src/foam/nanos/ruler/Ruled.js

```diff
@@ -39,11 +43,16 @@
       writePermissionRequired: true
     },
     {
-      class: 'foam.mlang.predicate.PredicateProperty',
+      class: 'FObjectProperty',
```

Comment:
curious why `PredicateProperty` is not working?

Reply:
Yes, we have class: 'foam..PredicateProperty' used numerous times elsewhere. 
do we have build warnings for each?

Reply:
It was overriden .. @jinn9  had a bug, looks like it wasn't saving or rendering properly in the UI.

Reply:
The predicate property in Rule.js didn't have class defined and this caused some issue when updating the predicate. I believe using PredicateProperty didn't resolve the issue thus FObjectProperty. 

 https://github.com/kgrgreer/foam3/pull/2075

Reply:
@jlhughes .. the warning wasn't for using PredicateProperty .. it was because the class changed on an inheriting class from PredicateProperty to FObjectProperty. So it shouldn't happen in other places.

Reply:
PredicateProperty used to work before, we've been using PredicateProperty other places besides here, do we want to investigate fixing PredicateProperty or update all those places to FObjectProperty instead? @jinn9 @jlhughes @kgrgreer